ui/app/src/components/PathNavigatorTree/PathNavigatorTree.tsx

```diff
@@ -241,8 +252,33 @@ export default function PathNavigatorTree(props: PathNavigatorTreeProps) {
       switch (type) {
         case itemsPasted.type:
         case folderCreated.type: {
-          if (payload.clipboard.type === 'CUT') {
-            debugger;
+          if (payload.clipboard?.type === 'CUT') {
```

Comment:
Confirm: it should go to else statement whenever is not a cut **including** paste items via copy?
Otherwise, this condition needs to be revisited and rather use `type` to make the determination.

Reply:
that is ok, it should go to the else statement for any operation excluding CUT, in this case COPY or folderCreated